docs/advanced/releases.md

```diff
@@ -0,0 +1,104 @@
+1. Update the `CHANGELOG.MD`. At the top you should have an `Unreleased` header with the `Added`, `Changed`, `Fixed` and `Removed` empty sections. After that, the header with the current version and link with the same beforementioned sections and a `Previous versions` header at the bottom that links to the previous minor release stable branch.
+
+  ```markdown
+  ## [Unreleased](https://github.com/decidim/decidim/tree/HEAD)
+
+  ### Added
+
+  ### Changed
+
+  ### Fixed
+
+  ### Removed
+
+  ## [v0.XX.0](https://github.com/decidim/decidim/releases/tag/v0.XX.0)
+
+  ### Added
+  ...
+
+  ## Previous versions
+
+  Please check [0.XX-stable](https://github.com/decidim/decidim/blob/0.XX-stable/CHANGELOG.md) for previous changes.
+  ```
+
+1. Push the changes `git add . && git commit -m "Bump develop to next release version" && git push origin develop`
+
+## Release Candidates
+
+Release Candidates are the same as beta versions. They should be ready to go to production, but publicly released just before in order to be widely tested.
+
+If this is a **Release Candidate version** release, the steps to follow are:
+
+1. Go to the release branch `git checkout release/x.y.z`.
+1. Update `.decidim-version` to the new version `x.y.z.rc1`
+1. Run `bin/rake update_versions`, this will update all references to the new version.
+1. Run `bin/rake bundle`, this will update all the `Gemfile.lock` files
+1. Run `bin/rake webpack`, this will update the JavaScript bundle.
+1. Commit all the changes: `git add . && git commit -m "Bump to rcXX version" && git push origin release/x.y.z`.
+1. Follow the link resulting from the previous command to create the PR for the new version.
+1. Usually, at this point, a deploy to meta-decidim is done during, at least, one week to validate the stability of the version.
+
+### During the validation period
+
+1. During the validation period bugfixes will go directly to the current `release/x.y.z` branch and ported to `develop`.
+1. During the validation period, translations to the officially supported languages must be added to Crowdin and when done, merged into `release/x.y.z`.
+
+### Major/Minor versions
+
+Once Release Candidates have been tested, merging into `master` after the validation period can be done.
+
+1. Go to the release branch `git checkout release/x.y.z`.
+1. Update `.decidim-version` by removing the `.rcN` suffix, leaving a clean version number like `x.y.z`
+1. Run `bin/rake update_versions`, this will update all references to the new version.
+1. Run `bin/rake bundle`, this will update all the `Gemfile.lock` files
+1. Run `bin/rake webpack`, this will update the JavaScript bundle.
+1. Commit all the changes: `git add . && git commit -m "Bump to v0.XX.0 final version" && git push origin release/x.y.z`.
+1. Follow the link resulting from the previous command to create the PR for the new version, but still don't merge it. The base for the PR should be `master`.
```

Comment:
```suggestion
1. Follow the link resulting from the previous command to create the PR for the new version in case it has not been created already, but still don't merge it. The base for the PR should be `master`.
```